tests/engines/registries/integration/index/index-test.ts

```diff
@@ -107,9 +107,9 @@ module('Registries | Integration | index', hooks => {
                     ),
                     expected: [
                         'link',
-                        'Index - Requirements and Roadmap',
+                        'Index - public roadmap',
                         // tslint:disable-next-line:max-line-length
```

Comment:
need to remove this line because the line is no longer too long

Reply:
Fixed, thank you!